.flowconfig

```diff
@@ -19,5 +19,5 @@ suppress_type=$FlowTODO
 module.name_mapper='.*react-docgen-loader.*' -> '<PROJECT_ROOT>/utils/emptyObject'
 module.name_mapper='.*.\(:?md\|svg\)$' -> '<PROJECT_ROOT>/utils/emptyString'
 module.name_mapper='^mineral-ui-icons\/?\(.*\)$' -> '<PROJECT_ROOT>/packages/mineral-ui-icons/src/\1'
-module.name_mapper='^mineral-ui\/?\(.*\)$' -> '<PROJECT_ROOT>/src/\1'
+module.name_mapper='^mineral-ui\/?\(.*\)$' -> '<PROJECT_ROOT>/src/library/\1'
```

Comment:
do we need another path for the website?

Reply:
Not at this time.  We would need it if we want to use shorthand directory aliases for our imports.

Reply:
That being said, would you like shorthand directory aliases for website and library?

Reply:
I think that would make our files cleaner & more straight-forward. If I remember correctly, @kylegach was thinking about doing this too.

Reply:
I think just the library alias is good for now.